packages/consumption/src/modules/attributes/local/LocalAttribute.ts

```diff
@@ -35,11 +35,13 @@ export interface ILocalAttribute extends ICoreSynchronizable {
     parentId?: ICoreId;
 }
 
-export type OwnSharedIdentityAttribute = LocalAttribute & { content: IdentityAttribute } & {
+export type OwnSharedIdentityAttribute = LocalAttribute & {
+    content: IdentityAttribute;
     shareInfo: LocalAttributeShareInfo;
 };
 
-export type OwnSharedRelationshipAttribute = LocalAttribute & { content: RelationshipAttribute } & {
+export type OwnSharedRelationshipAttribute = LocalAttribute & {
```

Comment:
Own shared relationship attributes never have the `shareInfo.sourceAttribute` field set. So its

```ts
export type OwnSharedRelationshipAttribute = LocalAttribute & {
    content: RelationshipAttribute;
    shareInfo: LocalAttributeShareInfo & { sourceAttribute: undefined };
};
```

Reply:
![image](https://github.com/nmshd/runtime/assets/35600954/e172d4d0-2519-42ab-838e-7c4abf6fbd2e)